src/modules/Login/index.tsx

```
@@ -1,24 +1,124 @@
+          message.error('验证码错误')
+          generateCaptcha()
+          form.setFieldValue('captcha', '')
+          return
+        }
+
+        // 这里添加实际的登录逻辑
+        await new Promise(resolve => setTimeout(resolve, 1000)) // 模拟请求
+        message.success('登录成功')
+        navigate('/index')
+      }
+      catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(unused-imports/no-unused-vars)